src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html

```diff
@@ -1,13 +1,12 @@
 <div>
-    <p>When running a job, Jenkins will use these credentials to check out the source code for builds. For Jenkins to do
-        this, it needs credentials with access to the projects and repositories you want it to build from.</p>
-
-    <p>To provide Jenkins with credentials you can:</p>
+    <p>When running a job, Jenkins requires credentials to checkout the source code for builds. To do this, it needs credentials with access to the projects and repositories you want it to build from.</p>
+    <p>You can provide Jenkins with credentials here by:</p>
     <ul>
-        <li>choose credentials from the list</li>
-        <li>add credentials as a <strong>Username with password</strong> (for the password, you can enter a Bitbucket Server password or a
+        <li>selecting credentials from the list</li>
+        <li>adding credentials as a <strong>Username with password</strong> (for the password, you can enter a Bitbucket Server password or a
             Bitbucket Server <a href="https://confluence.atlassian.com/x/a97-Nw">personal access token</a>)
         </li>
-        <li>Leave this empty to use default credentials if they exist.</li>
+        <li>leaving this empty to use default credentials if they exist</li>
     </ul>
+    <p>Alternatively, you can provide Jenkins with SSH credentials below. If you do this, Jenkins will use your SSH credentials instead by default.</p>
```

Comment:
We need to explicitly call out we are using SSH credentials for _clone operations only_. Users still need http for other actions with the plugin- this step is only optional if they have default creds or are accessing a public repo.

Reply:
Hi @mhenschke-atl . Thanks for the context. What about this?

```suggestion
    <p>In addition, you can provide Jenkins with SSH credentials below. If you do, Jenkins will use them for clone operations instead of the credentials you select here.</p>
```

Reply:
Yep this is good :)